pkg/apis/provisioning/v0alpha1/jobs.go

```diff
@@ -69,11 +69,11 @@ type JobSpec struct {
 	// Pull request options
 	PullRequest *PullRequestJobOptions `json:"pr,omitempty"`
 
-	// Required when the action is `export`
-	Export *ExportJobOptions `json:"export,omitempty"`
+	// Required when the action is `push`
+	Push *ExportJobOptions `json:"push,omitempty"`
```

Comment:
😬 we changed the action string to push/pull so must also change the options

Without this change, the frontend fails to compile.


Unrelated -- but here seems as good as anywhere